crates/flow-web/tests/web.rs

```diff
@@ -233,6 +233,42 @@ fn test_end_to_end_extend_read_bundle() {
     );
 }
 
+#[wasm_bindgen_test]
```

Comment:
For sure does not cover everything and is really more of a happy path test. Did not feel like writing every permutation with this or that property missing was necessary right now.